.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: python
+
+python:
+    - "2.7"
+
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get install -y openjdk-8-jre-headless
+
+install:
+    - wget https://wiki.dnanexus.com/images/files/dx-toolkit-current-ubuntu-14.04-amd64.tar.gz
+    - tar xf dx-toolkit-current-ubuntu-14.04-amd64.tar.gz 
+    - source dx-toolkit/environment
```

Comment:
Do we definitely need dx-toolkit and its environment to run the jar? Might be preferable to install the minimum that we need with pip or apt-get: https://wiki.dnanexus.com/Downloads#Alternative-downloads. The tarball is a pretty opaque set of dependencies.

Reply:
I think it might work with just pip installing `dxpy`; I'll test it. The docker point is interesting because we need to think about how to encapsulate the "stage" and "compile" behavior that's likely going to be required for running external tests. Putting it into a docker image might be the best solution for that.